src/django/sqls/0150_revert_add_percent_to_number.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS add_percent_to_number(int8, int8);
```

Comment:
If you want to delete this function, you should use the same parameter types that were set when declaring the function.

Reply:
changed